src/clsourcedependencies_parser.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#if NINJA_ENABLE_CL_SOURCE_DEPENDENCIES
```

Comment:
nit: move this inside the `#ifndef / #define  ... #endif` block (this helps the compiler recognizing the guard pattern).

Reply:
I dropped whole-file `#if` conditions.